setup.py

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python
```

Comment:
I'm not fan of executable setup.py. The usual command is `python setup.py`, not `./setup.py`